widget/table.go

```diff
@@ -556,12 +569,12 @@ func (r *tableCellsRenderer) Refresh() {
 				if c == nil {
 					continue
 				}
-
-				c.Move(fyne.NewPos(theme.Padding()+cellOffset,
-					theme.Padding()+float32(row)*(r.cells.cellSize.Height+separatorThickness)))
-				c.Resize(fyne.NewSize(colWidth-theme.Padding()*2, r.cells.cellSize.Height-theme.Padding()*2))
 			}
 
+			c.Move(fyne.NewPos(theme.Padding()+cellOffset,
+				theme.Padding()+float32(row)*(r.cells.cellSize.Height+separatorThickness)))
+			c.Resize(fyne.NewSize(colWidth-theme.Padding()*2, r.cells.cellSize.Height-theme.Padding()*2))
```

Comment:
`Resize` can call `Layout`, so I wonder if it would be better to have this after the `updateCell` call, WDYT?

Reply:
Is there a difference to moving around widgets that contain template data compared to real data?
In a way it felt better to call update with the dimensions set, in case they matter

Reply:
The prior art here is tree.go#465, so whichever approach is picked should be consistent across the collections

Reply:
That doesn't answer my questions, sorry. I would rather do the right thing rather than copy what is elsewhere.
If they both need to update I could do that.

Reply:
The dev-defined-canvasobject should not need to be resized before being updated, but there certainly isn't any promise in the public API either way so it shouldn't hold up the review